azure/storage-account-sample/src/main/java/com/yanchware/fractal/azure/sample/components/StorageAccountComponent.java

```diff
@@ -0,0 +1,29 @@
+package com.yanchware.fractal.azure.sample.components;
+
+import com.yanchware.fractal.sdk.domain.entities.livesystem.paas.providers.azure.storageaccount.AzureStorageAccount;
+import com.yanchware.fractal.sdk.domain.entities.livesystem.paas.providers.azure.storageaccount.AzureStorageAccountBackup;
+import com.yanchware.fractal.sdk.domain.entities.livesystem.paas.providers.azure.storageaccount.AzureStorageAccountInfrastructure;
+import com.yanchware.fractal.sdk.domain.entities.livesystem.paas.providers.azure.storageaccount.valueobjects.AzureStorageSkuName;
+
+public class StorageAccountComponent {
+
+  public static AzureStorageAccount getStorageAccountComponent(String id) {
+
+    // Backup policy definition:
+    final String backupPolicy = "{\"workLoadType\":\"AzureFileShare\",\"backupManagementType\":\"AzureStorage\",\"timeZone\":\"UTC\",\"schedulePolicy\":{\"schedulePolicyType\":\"SimpleSchedulePolicy\",\"scheduleRunFrequency\":\"Daily\",\"scheduleRunTimes\":[\"2018-01-24T01:00:00.000000000Z\"]},\"retentionPolicy\":{\"retentionPolicyType\":\"LongTermRetentionPolicy\",\"dailySchedule\":{\"retentionDuration\":{\"count\":30,\"durationType\":\"Days\"},\"retentionTimes\":[\"2018-01-24T01:00:00.000000000Z\"]},\"weeklySchedule\":null,\"monthlySchedule\":{\"retentionScheduleFormatType\":\"Daily\",\"retentionScheduleDaily\":{\"daysOfTheMonth\":[{\"date\":1,\"isLast\":false}]},\"retentionDuration\":{\"count\":12,\"durationType\":\"Months\"},\"retentionScheduleWeekly\":null,\"retentionTimes\":[\"2018-01-24T01:00:00.000000000Z\"]},\"yearlySchedule\":null}}";
```

Comment:
Could we maybe use a different way to display this? To see it clearly? Like

```
public String textBlocks() {
    return """
        Get busy living
        or
        get busy dying.
        --Stephen King""";
}
```

Reply:
Sure, good idea!